src/routes/index.js

```diff
@@ -0,0 +1,24 @@
+import Home from '../pages/Home.js';
+import Blog from '../pages/Blog.js';
+import Collaborator from '../pages/Collaborator.js';
+import Error404 from '../pages/Error404.js';
+
+const routes = {
+	'/': Home,
+	'/blog': Blog,
+	'/collaborator': Collaborator,
+};
+
+const router = () => {
+	const $main = document.getElementById('main');
```

Comment:
Can we use `document.querySelector()? instead of getElement?` i mean, it's more readable and efficient because we can get class and Id with that function.
I think it's better, what about you?

Reply:
Read my comment

Reply:
Rene getquery selector is like 1000% slower than getElementById, but is true that if create a simple app  it should not damage to much the performance =P

Reply:
Understood, we won't use `document.querySelector()` i thought i doesn't affect the web application performance